applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -392,11 +409,14 @@ static bool event_handler(const struct event_header *eh)
 			cast_ble_peer_event(eh);
 
 		if (event->state == PEER_STATE_DISCONNECTED) {
-			for (size_t i = 0; i < ARRAY_SIZE(hidc); i++) {
-				if ((bt_gatt_hids_c_assign_check(&hidc[i])) &&
-				    (bt_gatt_hids_c_conn(&hidc[i]) == event->id)) {
+			for (size_t i = 0; i < ARRAY_SIZE(subscribers); i++) {
+				if ((bt_gatt_hids_c_assign_check(
+				      &subscribers[i].hidc)) &&
```

Comment:
please do not break the lines as it is barely readable

Reply:
ok, I will changeit back. I did this to fit in 80 characters limit